addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/RequestorJob.java

```diff
@@ -76,20 +76,22 @@ public void verifyParameters(AutomationProgress progress) {
 
         this.verifyUser(this.getParameters().getUser(), progress);
 
-        Object o = jobData.get(REQUESTS);
-        if (o == null) {
+        Object requestsList = jobData.get(REQUESTS);
+        if (requestsList == null) {
             return;
         }
-        if (!(o instanceof ArrayList<?>)) {
-            progress.error(Constant.messages.getString("automation.error.requestor.badlist", o));
+        if (!(requestsList instanceof ArrayList<?>)) {
```

Comment:
probably it should be List<?>.. rather than ? you should specify the type that's expected maybe? :)